cookienotice/cookienotice.js

```diff
@@ -0,0 +1,37 @@
+window.addEventListener("load", function () {
+	var cookiename = 'cncookiesaccepted'
+	var cookie = getCookie(cookiename);
+
+	if (cookie == "") {
+		document.getElementById('cookienotice-box').style.display = 'block';
+		document.getElementById('cookienotice-ok-button').onclick = function () {
+			console.log('clicked');
```

Comment:
Please remove debug code from committed code.

Reply:
no problem with second pass. I am still learning.

done

Reply:
The `console.log` call is still present in the last version of this PR.